src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -267,7 +267,7 @@ public GetItemResult getItem(GetItemRequest getItemRequest) {
         getItemRequest.withTableName(tableMapping.getPhysicalTable().getTableName());
 
         // map key
-        getItemRequest.setKey(tableMapping.getItemMapper().apply(getItemRequest.getKey()));
+        getItemRequest.setKey(tableMapping.getKeyMapper().apply(getItemRequest.getKey()));
 
```

Comment:
This the only runtime change, only the key fields of the table are mapped, not all fields in the key.

Reply:
If I flip `getKeyMapper()` to `getItemMapper()` then `GetTest.getHkRk_TableWithGsiHkSameAsTableRk` fails with `The number of conditions on the keys is invalid`.